layouts/index.hbs

```diff
@@ -42,6 +42,7 @@
         </div>
 
         {{#if project.latestVersions.current.node}}
+        {{#if project.latestVersions.current.hideCurrent}}
```

Comment:
do you mean this to render when `hideCurrent` is true? or the inverse, when hideCurrent is false?

Reply:
fixed.